packages/mask/src/extension/popups/pages/Friends/ContactCard/ConnectedAccounts/index.tsx

```diff
@@ -0,0 +1,107 @@
+import { memo, useState } from 'react'
+import { makeStyles } from '@masknet/theme'
+import { Box, Typography, useTheme, ButtonBase } from '@mui/material'
+import { type BindingProof, PopupRoutes, NextIDPlatform } from '@masknet/shared-base'
+import { PersonaContext } from '@masknet/shared'
+import { useNavigate } from 'react-router-dom'
+import { TwitterAccount } from '../TwitterAccount/index.js'
+import { Account } from '../Account/index.js'
+import { safeUnreachable } from '@masknet/kit'
+
+const useStyles = makeStyles()((theme) => ({
+    connectedAccounts: {
+        borderBottomLeftRadius: '6px',
+        borderBottomRightRadius: '6px',
+        background: theme.palette.maskColor.white,
+        padding: '8px',
+        position: 'relative',
+    },
+    more: {
+        display: 'flex',
+        alignItems: 'center',
+        justifyContent: 'center',
+        width: '24px',
+        height: '24px',
+        background: 'rgba(28, 104, 243, 0.1)',
+        borderRadius: '50%',
+        position: 'absolute',
+        right: '10px',
+    },
+}))
+
+interface ConnectedAccountsProps {
+    avatar?: string
+    profiles: BindingProof[]
+    nextId: string
+    publicKey?: string
+    isLocal?: boolean
+}
+
+export const ConnectedAccounts = memo<ConnectedAccountsProps>(({ avatar, nextId, profiles, publicKey, isLocal }) => {
```

Comment:
We prefer named function, or setting `displayName` implicit.
BTW, recommend to take a look at this presentation about react-dev-tools.


## Named function

```suggestion
export const ConnectedAccounts = memo<ConnectedAccountsProps>(function ConnectedAccounts({ avatar, nextId, profiles, publicKey, isLocal }) {
```

## Or set `displayName` 
```ts
ConnectedAccounts.displayName = 'ConnectedAccounts'
```

Reply:
[this presentation](https://docs.google.com/presentation/d/1o2jfOd5p6Kcs-wix1AuA6M_Gu7kZ9650uw8CYCvCCUU/edit#slide=id.g2241b2f7699_0_14)